.pre-commit-config.yaml

```
@@ -0,0 +1,18 @@
+repos:
+  - repo: https://github.com/lycheeverse/lychee.git
+    rev: v0.15.1
+    hooks:
+      - id: lychee
+        # Optionally include additional CLI arguments
+        args: ["--no-progress", "--exclude", "file://"]
+
+repos:
```

yamllint (1.35.1)

[error] duplication of key "repos" in mapping

(key-duplicates)